src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -316,6 +316,19 @@ public async Task CancelExport(Uri contentLocation, CancellationToken cancellati
             await HttpClient.SendAsync(message, cancellationToken);
         }
 
+        public async Task<string> DataConvertAsync(Parameters parameters, CancellationToken cancellationToken = default)
+        {
+            string requestPath = "$data-convert";
+            using var message = new HttpRequestMessage(HttpMethod.Post, requestPath);
+            message.Content = new StringContent(parameters.ToJson(), Encoding.UTF8, "application/json");
```

Comment:
Just want to note what over Post requests build content with `CreateStringContent` which utilize `_contentType` property and allow you test not only json, but also xml.

Reply:
Thanks. That's really a good idea